dist/backend/web-gl/function-node.js

```diff
@@ -652,17 +683,64 @@ module.exports = function (_FunctionNodeBase) {
 				}
 				var retDeclaration = [];
 				this.astGeneric(declaration, retDeclaration);
-				if (retDeclaration[2] === 'getImage2D(' || retDeclaration[2] === 'getImage3D(') {
-					if (i === 0) {
-						retArr.push('vec4 ');
+				var declarationType = 'Number';
+				if (i === 0) {
+					var init = declaration.init;
+					if (init) {
+						if (init.object) {
+							if (init.object.type === 'MemberExpression' && init.object.object) {
+								if (init.object.object.type === 'Identifier') {
+									var _type2 = this.getParamType(init.object.object.name);
+									declarationType = typeLookupMap[_type2];
+									if (!declarationType) {
+										throw new Error('unknown lookup type ' + typeLookupMap);
+									}
+									debugger;
```

Comment:
Why we have 'debugger' statements in public code? May be we should remove it? (it exists in lines: 698, 705, 713, 720... and maybe somewhere else...